subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -43,6 +43,14 @@ config BT_EATT_MAX
 	help
 	  Number of Enhanced ATT bearers available.
 
+
+config BT_EATT_AUTO_CONNECT
+	bool "Automatically connect EATT bearers when a link is established"
```

Comment:
I think we need to decide between EATT bearers or EATT channels, and use the same term everywhere.

Reply:
That is because the spec doesn't differentiate an EATT bearer to an ATT bearer, ATT bearer are uniquely identified by its L2CAP CID and thus can save state (CCC states, etc), EATT bearer uses L2CAP Connection Oriented Channels and cannot save state thus why channels was used instead, though this explanation could probably be added as part of the documentation.